index.html

```diff
@@ -27,15 +27,17 @@ <h1>Medicina Emocional <br> para tu vida</h1>
     <section id="Fan-de-las-Flores">
```

Comment:
los **id's** siempre empiezan en minúsculas y no se dividen con guiones se usa CamelCase: ej **fanDeLasFlores**

Reply:
This adjusted is solved in next commit of branch feature/body